packages/wallet/backend/migrations/20230620082429_add_publicKey_to_payment_pointers.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('paymentPointers', (table) => {
+    table.jsonb('keyIds')
```

Comment:
As an alternative, this could be an array: packages/wallet/backend/migrations/20230620082429_add_publicKey_to_payment_pointers.js

Reply:
Well, I think only jsonb is supported in knex.